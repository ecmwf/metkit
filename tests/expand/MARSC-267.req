retrieve,area=40/-180/-40/180,class=s2,date=2024-01-11,expver=prod,levtype=pl,padding=0,param=130,step=0/to/1104/by/24,stream=enfh,time=00:00:00,type=pf,levelist=200/300/500/700/850/925/1000,number=1,origin=ecmf,hdate=2004-01-11,target=target.out