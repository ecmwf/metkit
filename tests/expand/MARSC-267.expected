RETRIEVE,
    CLASS      = S2,
    TYPE       = PF,
    STREAM     = ENFH,
    EXPVER     = prod,
    REPRES     = SH,
    LEVTYPE    = PL,
    LEVELIST   = 200/300/500/700/850/925/1000,
    PARAM      = 130,
    DATE       = 20240111,
    HDATE      = 20040111,
    TIME       = 0000,
    STEP       = 0/24/48/72/96/120/144/168/192/216/240/264/288/312/336/360/384/408/432/456/480/504/528/552/576/600/624/648/672/696/720/744/768/792/816/840/864/888/912/936/960/984/1008/1032/1056/1080/1104,
    NUMBER     = 1,
    ORIGIN     = ECMF,
    TARGET     = target.out,
    AREA       = 40/-180/-40/180,
    PADDING    = 0